gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
+            ],
+            karamaConf: 'karma.debug.conf.js'
+        };
+
+    // starts up Karma using Chrome in debug mode
+    function runKarma(configFile, cb) {
```

Comment:
I just copied the default Karma file, but I hate abbreviated names so I'll see if I can find out what it is.